packages/appChrome/style.ts

```diff
@@ -88,8 +88,6 @@ export const sidebarContainer = (theme: AppChromeTheme, isOpen?: boolean) => {
   `;
 };
 
-// TODO: replace animation duration/easing with design tokens
-// once design has agreed on animation
 export const sidebarAnimator = css`
```

Comment:
We have no plans for animation design tokens. I think we can consider this in the future, but for now it is not relevant.